client/src/components/analyze-name/existing-nr-display.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <v-form @submit="submitName">
+    <v-text-field v-model="nameSearch"
+                  solo
+                  flat
+                  placeholder="Enter a Name or your Request Number (eg NR123456)"
+                  id="analyze-name-text-field">
+      <template v-slot:append>
+        <v-icon class="name-search-icon">search</v-icon>
+      </template>
+    </v-text-field>
+  </v-form>
+</template>
+
+<script lang="ts">
+  import { Component, Vue } from 'vue-property-decorator'
+  import analyzeName from '@/store/analyze-name-module'
+
+  @Component
+  export default class ExisitingNRDisplay extends Vue {
+    private get nameSearch(): string {
+      return analyzeName.analyzedName
+    }
+
+    private set nameSearch(name: string) {
+      analyzeName.captureName(name)
+    }
+
+    private submitName() {
+
+    }
+  }
+</script>
+
+<style lang="stylus" scoped>
+  .name-search-icon
+    color #1976d2 !important
```

Comment:
You can use CSS variables for colour, here, if you first import your theme file (in your case, `variables.styl`).